src/lib/eliom_client.client.ml

```diff
@@ -715,10 +724,10 @@ let set_template_content ~replace ~uri ?fragment =
     run_onunload_wrapper (really_set content) cancel
 
 let set_uri ~replace ?fragment uri =
-  (* Changing url: *)
+  let advance = not replace in
```

Comment:
I don't understand this. Does this mean that, if we call `change_page` with argument `~replace:true`, then the new page will never become active?